dspace-api/metadataExportSearch.csv

```diff
@@ -0,0 +1,3 @@
+id,collection,dc.date.accessioned,dc.date.available,dc.description.provenance[en],dc.identifier.uri,dc.subject[*],dc.title[*]
```

Comment:
Is this file included by accident?  If this file is needed for tests, it should be placed somewhere under `/src/test/resources/`.  Otherwise, it should just be deleted.

Reply:
This has been deleted

Reply:
Since this is causing the ITs to fail, we will move it instead